Packs/PANOSPolicyOptimizer/ReleaseNotes/1_0_4.md

```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### PAN-OS Policy Optimizer
+- Documentation and metadata improvements.
```

Comment:
Why do we need RN and version bump? it will trigger users for a change in a pack without having any real change. If this is due to SDK validate error it's a dev bug.

Reply:
Will it reflect in the marketplace front in XSOAR if I will not bump the version?

Reply:
Do you mean would it be updated in the pack details when clicked? yes

Reply:
ok, so RMing